src/data/structs.rs

```diff
@@ -205,8 +206,7 @@ pub struct ValidateRequest {
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct AcceptRequest {
     pub consignment: String,
-    // pub outpoint: OutPoint,
-    // pub blinding_factor: String,
+    pub reveal: Option<Reveal>,
```

Comment:
Let's make this a non-optional String (since accepting without a reveal doesn't make sense, right?)

Also, as a string, it can be parsed and encoded using these formatting and parsing methods:
https://github.com/diba-io/bitmask-core/pull/109/files#diff-d9c6a42fed07589e989e8da734d4ba5f8ce37f6cf4cd63b3a22906f6b9451d98R71